pca_reduction.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+import numpy as np
+#import sys
+#sys.path.insert(1,r"C:\Users\Yakoubi\regression_pipeline")
+from data_preparation import *
+from sklearn.decomposition import PCA
+
+
+def reduce_dimensionality(dataset: pd.DataFrame):
```

Comment:
Add a return hint (ie: what type of object your method returns). Here you should add: `-> pd.DataFrame`.